src/main/java/org/spongepowered/api/entity/living/Sittable.java

```diff
@@ -42,5 +42,4 @@
      * @param sitting Whether this entity is sitting
      */
     void setSitting(boolean sitting);
-
```

Comment:
Actually, per the style guide they're [completely optional](https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s4.6.1-vertical-whitespace). So it's best not to fret about these and just leave them however they happen to be in order to reduce diff pollution.